src/librbd/operation/ResizeRequest.cc

```diff
@@ -401,9 +401,9 @@ void ResizeRequest<I>::send_update_header() {
   librados::ObjectWriteOperation op;
   if (image_ctx.old_format) {
     // rewrite only the size field of the header
-    // NOTE: format 1 image headers are not stored in fixed endian format
```

Comment:
@dillaman @idryomov can you check this? maybe the kernel does the endian swap and librbd did not?

Reply:
The kernel expects the size in LE and does the swap, so this change looks fine to me.

Reply:
Agreed that it's fine -- librbd never did the swap before hence the comment (i.e. it's known to be broken but it's also a deprecated/disabled V1 image format issue).